ivre/db/mongo.py

```diff
@@ -721,6 +721,52 @@ def getscreenshot(port):
         if url == "field":
             return port.get('screendata')
 
+    def set_label(self, flt, group, label, archives=False):
```

Comment:
I don't really like this function, mainly because you will run an `.update()` even when the record has not been changed.